ac/ac-ranking/src/ActivityRunner/index.js

```diff
@@ -146,6 +148,7 @@ const ActivityRunner = (props: ActivityRunnerT) => {
             <hr style={{ height: '5px' }} />
             <div>
               <div style={{ width: '100%' }}>
+                <p>At rank {nKey(answers[userInfo.id] || {}) + 1}, add item:</p>
```

Comment:
After I have selected everything it still says "At rank 5, add item:" and then no more buttons. This should disappear when everything is in the list.

Reply:
Fixed by houshuang